### PR TITLE
feat(replays): add resources menu to replay details

### PR DESCRIPTION
**Pull request by michellewzhang**
- Add resource docs hovercard to replay details 
- Does not appear on full screen
- Based on [this design](https://www.figma.com/file/eOC1hwNbHMzFhJsZGkU3UH/Specs%3A-Touchpoints-for-Configuration-Resources?type=design&node-id=0-1&mode=design&t=Mdq8Z9K9PWiKSLOG-0)

https://github.com/getsentry/sentry/assets/56095982/280746e3-34d4-42fc-85a6-6e4b2cae83a9

Closes https://github.com/getsentry/team-replay/issues/323

### CODE REVIEW

**Comment by ryan953** [comment]
There's a lot of padding around the items: 
SCR-20240122-legk

File: static/app/components/replays/replayPlayer.tsx [hunk 1]
```diff
@@ -113,13 +120,69 @@ function BasePlayerRoot({className, isPreview = false}: Props) {
...
+  }
+
   return (
     <NegativeSpaceContainer ref={windowEl} className="sentry-block">
       <div ref={viewEl} className={className} />
       {fastForwardSpeed ? <PositionedFastForward speed={fastForwardSpeed} /> : null}
       {isBuffering ? <PositionedBuffering /> : null}
       {isPreview ? null : <PlayerDOMAlert />}
       {isFetching ? <PositionedLoadingIndicator /> : null}
+      {!(isFullscreen || isPreview) && <ResourceCard />}
```
Comment (ryan953) [inline static/app/components/replays/replayPlayer.tsx:185]: `isPreview` nice catch!

File: static/app/components/replays/replayPlayer.tsx [hunk 2]
```diff
@@ -253,4 +316,40 @@ const SentryPlayerRoot = styled(PlayerRoot)`
   }
 `;
 
+const ResourceCardContainer = styled('div')`
+  position: absolute;
+  bottom: ${space(1)};
+  right: 1%;
```
Comment (ryan953) [inline static/app/components/replays/replayPlayer.tsx:322]: Can we make this a fixed amount? i'm gussing at a size, but 1% seems funny to me
```suggestion
  right: ${space(1)};
```

here's 1% when the container is different widths:
| wide container | narrow container |
| --- | --- |
| SCR-20240122-lgaa | SCR-20240122-lgaq

**Comment by michellewzhang** [comment]
> There's a lot of padding around the items: SCR-20240122-legk

fixed

File: static/app/components/replays/replayPlayer.tsx [hunk 3]
```diff
@@ -113,13 +121,76 @@ function BasePlayerRoot({className, isPreview = false}: Props) {
...
+          {({css}) => (
+            <Hovercard
+              body={<ResourceButtons />}
+              bodyClassName={css`
+                padding: ${space(1)};
+              `}
+              header={t('Documentation Resources')}
+              position="top-end"
+            >
+              <Button icon={<IconQuestion />} aria-label="replay resources" />
```
Comment (ryan953) [inline static/app/components/replays/replayPlayer.tsx:178]:
```suggestion
              <Button icon={<IconQuestion />} aria-label={t('replay resources')} />
```

**Review by ryan953 — APPROVED** [review]

**Pull request merged**